execution/engine/testdata/complex_nesting_query_with_art.json

```diff
@@ -430,7 +430,6 @@
                                 "__typename": "Review",
                                 "attachments": [
                                   {
-                                    "__typename": "Question",
```

Comment:
why typenames are removed?

Reply:
ah, disregard

I see now

Reply:
it was duplicate coming from subgraph, now deduplicated (to generate valid JSON) by trace deduplication